package.json

```diff
@@ -149,6 +150,7 @@
     "nyc": "^15.1.0",
     "playwright": "^1.21.1",
     "prettier": "^2.7.1",
+    "process": "^0.11.10",
```

Comment:
Required as we were relying on it, with a transitive dependency of docsearch.js.